mattermost/mattermost-plugin-gitlab
# [MM-18640] Update mm redux for csrf

#### Summary

This PR updates the `mattermost-redux` dependency to handle potential csrf issues.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-18640

#### QA Steps

- Set `ServiceSettings.ExperimentalStrictCSRFEnforcement` to true in the server's config.json
- Verify that when you click on a connected user's profile picture, they have a user attribute linking to their gitlab account.
